src/lua/compat.c

```diff
@@ -5,8 +5,40 @@
  */
 
 #include <lua/compat.h>
+#include "serializer.h"
+
+int
+lbox_msgpuck_escape_forward_slash_toggle(struct lua_State *L)
+{
+	assert(lua_isboolean(L, -1) && "boolean argument expected");
+	bool esc_slash = lua_toboolean(L, -1);
+	lua_pop(L, 1);
+
+	if (esc_slash)
+		mp_char2escape['/'] = "\\/";
+	else
+		mp_char2escape['/'] = NULL;
+
+	return 0;
+}
+
+int
+lbox_json_escape_forward_slash_toggle(struct lua_State *L)
+{
+	assert(lua_isboolean(L, -1) && "boolean argument expected");
+	bool esc_slash = lua_toboolean(L, -1);
+	lua_pop(L, 1);
+
+	if (serializer_set_option_default("encode_escape_forward_slash", esc_slash))
+		assert(!"There is no encode_escape_forward_slash option"
+			" in serializer defaults.");
+
+	return 0;
+}
 
 static const struct luaL_Reg internal_compat[] = {
+	{"msgpuck_escape_forward_slash_toggle", lbox_msgpuck_escape_forward_slash_toggle},
+	{"json_escape_forward_slash_toggle", lbox_json_escape_forward_slash_toggle},
```

Comment:
The line length is more than 80 characters.

Reply:
Fixed.